src/storage/page_buffer.c

```diff
@@ -7836,6 +7841,34 @@ pgbuf_claim_bcb_for_fix (THREAD_ENTRY * thread_p, const VPID * vpid, PAGE_FETCH_
   return bufptr;
 }
 
+/*
+ * pgbuf_request_data_page_from_page_server () - Sends a request for a page to Page Server.
+ *   return: void
+ */
+static void
+pgbuf_request_data_page_from_page_server (const VPID * vpid)
+{
+#if defined (SERVER_MODE)
+  /* Send a request to Page Server for the Page. */
+  if (get_server_type () == SERVER_TYPE_TRANSACTION)
+    {
+      constexpr size_t INT32_SIZE = 4;
```

Comment:
This is C++ code in a .c file and indent does not know to properly handle it. Add // *INDENT-OFF* at the start of the scope & // *INDENT-ON* at the end of the scope and fix indentation manually.